frontend/scripts/react-components/table-modal/table-modal.component.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Heading from 'react-components/shared/heading/heading.component';
+import Button from 'react-components/shared/button';
+import Table from 'react-components/dashboard-element/dashboard-widget/table-modal/table';
+import 'react-components/dashboard-element/dashboard-widget/table-modal/table-modal.scss';
```

Comment:
This file is not needed right?

Reply:
The SCSS? We need it

Reply:
Nope sorry, the whole `react-components/table-modal` directory.

Reply:
Is true 🤦‍♂️